README.md

```diff
@@ -0,0 +1,34 @@
+# grapejuice-flatpak
+## Installation
+### Step 1
+Download the GJTestRepo locally by running the following commands
+```bash
+git clone https://github.com/Thelolguy1/grapejuice-flatpak.git
+cd grapejuice-flatpak
+```
+### Step 2
+Add the flathub and the GJTestRepo
+```bash
+flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+flatpak --user remote-add --no-gpg-verify grapejuiceTest GJTestRepo
+```
+### Step 3
+Finally install the grapejuice flatpak
+```bash
+flatpak --user install grapejuiceTest com.gitlab.brinkervii.grapejuice
+```
+_If you want a global installation remove the '--user' flag from Step 2/3_
+
+## Simplified installation
+You can run ./install.sh and ./uninstall.sh for an automated install
+
+## Build It Yourself
+To build the flatpak by yourself, run
+1. build.sh (generates build-dir)
+2. run_bash.sh and verify that the container works.
+3. export.sh (generates GJTestRepo)
+
+## Disclaimer
+PLEASE NOTE: ROBLOX PLAYER DOES NOT FUNCTION YET, DUE TO ISSUES WITH VMPROTECT
```

Comment:
Not true, this will work with the flatpak:

**app/org.winehq.Wine/x86_64/stable-21.08**

Reply:
oh sorry about that, i copied that part from the existing readme

Reply:
Nevermind, I can handle it